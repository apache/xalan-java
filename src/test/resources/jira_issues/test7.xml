<data>
  <a>mnopq</a>
  <b>abcde 234</b>
  <c>abcde 567</c>
</data>
