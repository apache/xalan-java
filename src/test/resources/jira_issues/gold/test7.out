<?xml version="1.0" encoding="UTF-8"?><set>
  <problem>
    <question>Which lots have red cars?</question>
    <answer>
      <lot id="L"/>
      <lot id="L"/>
      <lot id="N"/>
    </answer>
  </problem>
</set>
